fegrow/receptor.py

```diff
@@ -28,22 +35,71 @@
 class NoPostMinimisationConformersError(Exception):
     """Raise if no conformers survive minimisation (due to e.g. simulation blowing up)"""
 
+def chimera_path_check():
+    #check if chimera is in the path, if not, raise an error
+    if not shutil.which("chimera"):
+        raise EnvironmentError("Chimera is not in the PATH. Please install Chimera and ensure it is accessible from the command line.")
+
+def chimera_protonate(input_file: str, output_file: str, verbose: bool = False):
+    """
+    Use Chimera to protonate the receptor.
+
+    :param input_file: The name of the pdb file which contains the receptor.
+    :param output_file: The name of the pdb file the fixed receptor should be wrote to.
+    :param pH:The ph the pronation state should be fixed for.
+    :param verbose: If True, print the Chimera output.
+    """
+    chimera_path_check()
+    #from chimera import runCommand as rc
+
+    # Silence stdout if not verbose
+    with open(os.devnull, 'w') as devnull:
+        if verbose:
+            stdout_target = None
```

Comment:
these do not seem to be used, initially I would not recommend silencing. Please let me know if there is a specific issue you're trying to address here